lhotse/features/librosa_fbank.py

```diff
@@ -0,0 +1,110 @@
+from dataclasses import asdict, dataclass
+
+import numpy as np
+from lhotse import FeatureExtractor
+from lhotse.features.base import register_extractor
+from lhotse.utils import Seconds, compute_num_frames, is_module_available
+
+
+if is_module_available("librosa"):
```

Comment:
I'd put this thing inside `logmelfilterbank` function instead, so that the class can be visible in global `lhotse` namespace without import errors, and only throws the error if somebody uses it without installing librosa

Reply:
Done